google-cloud-storage/src/main/java/com/google/cloud/storage/StorageException.java

```diff
@@ -73,4 +82,25 @@ public static StorageException translateAndThrow(RetryHelperException ex) {
     BaseServiceException.translate(ex);
     throw new StorageException(UNKNOWN_CODE, ex.getMessage(), ex.getCause());
   }
+
+  /**
+   * Translate IOException to the StorageException that caused the error. This method defaults to
```

Comment:
to the... --> to a StorageException representing the cause of the error. (otherwise you have the effect coming before the cause.)